cumulusci/salesforce_api/package_zip.py

```diff
@@ -95,6 +98,7 @@ def __init__(self, namespace, version, activateRSS=False, password=None):
         self.version = version
         self.activateRSS = activateRSS
         self.password = password
+        self.securityType = securityType
```

Comment:
Worth validating and throwing a `TaskOptionsError` just to not start an operation we know will fail?

Reply:
Good point re validation, but task options should really be validated in _init_options so that errors are thrown before the flow starts, not in code called by _run_task.